tests/components/test_smoothing.cpp

```diff
@@ -29,15 +35,15 @@ TEST_CASE("smoothing_Newton_Method")
         scheduler.add_operation_type<operations::tri_mesh::VertexSmoothUsingDifferentiableEnergy>(
             "optimize_vertices",
             std::move(op_settings));
-    Tuple tuple = mesh.face_tuple_from_vids(2, 4, 5);
-    spdlog::warn("Initial valuenorm: {}", factory.settings().energy->get_one_ring_value(tuple));
+    Tuple tuple = mesh.face_tuple_from_vids(0, 1, 0);
+    spdlog::warn("Initial valuenorm: {}", factory.settings().energy->get_value(tuple));
```

Comment:
I am not a big fan of logging output in tests. They make the test output very long and I think they should be only here for debugging but removed when the test is finished.

Reply:
We should set them to trace